api/database/seeders/JobPosterTemplateSeeder.data.json

```diff
@@ -0,0 +1,1699 @@
+{
+  "data": {
+    "jobPosterTemplates": [
+      {
+        "referenceId": "IT-03-03-SU-012",
+        "name": {
+          "en": "Team Leader - IT Planning and Reporting ",
```

Comment:
Could you strip out leading and trailing spaces, like on this line?

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11523/commits/d9848879662c95abe227f1c4a9c39431356f51e4